tests/post/steps/test_volume.py

```diff
@@ -12,7 +13,6 @@
 
 # Constants {{{
 
-
```

Comment:
Why this change?
If you really want to to it, should be done in the previous commit.